src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -43,42 +57,112 @@ public Intake(int motorId, int frontPiston, int backPiston) {
             .i(0.0)
             .d(0.0)
             .positionWrappingEnabled(true);
-        intakeMotor.configure(intakeConfig, ResetMode.kNoResetSafeParameters, PersistMode.kPersistParameters);
 
-        intakeEncoder = intakeMotor.getEncoder();
-        intakeController = intakeMotor.getClosedLoopController();
-        intakeEncoder.setPosition(0);
+        grabMotor.configure(intakeConfig, ResetMode.kNoResetSafeParameters, PersistMode.kPersistParameters);
+        grabEncoder = grabMotor.getEncoder();
+        grabController = grabMotor.getClosedLoopController();
+        grabEncoder.setPosition(0);
```

Comment:
Tady bude potřeba udělat nějakou "reset" funkci, protože když deployneme kód a intake bude v náhodný pozici, tak se pak bude točit divně